tests/acceptance/features/apiSharingNgShareInvitation/shareInvitations.feature

```diff
@@ -3198,3 +3198,39 @@ Feature: Send a sharing invitations
     Then the HTTP status code should be "200"
     And for user "Brian" the space Shares should contain these entries:
       | textfile.txt |
+
+  @issue-9642
+  Scenario Outline: list multiple shares
```

Comment:
Also, this scenario should go to `apiSharingNg1/sharedWithMe.feature` file

Reply:
okay. sure!